test/csuite/CMakeLists.txt

```diff
@@ -85,14 +85,18 @@ define_c_test(
     ARGUMENTS -h $<SHELL_PATH:$<TARGET_FILE_DIR:test_scope>/WT_HOME>
 )
 
-define_c_test(
-    TARGET test_tiered_abort
-    SOURCES tiered_abort/main.c
-    DIR_NAME tiered_abort
-    EXEC_SCRIPT ${CMAKE_CURRENT_SOURCE_DIR}/tiered_abort/smoke.sh
-    ARGUMENTS -b $<TARGET_FILE:test_tiered_abort>
-    DEPENDS "WT_POSIX"
-)
+# define_c_test(
```

Comment:
Whilst this is completely commented out, I'd prefer we add the following underneath it:
```
if (WT_POSIX)    
    create_test_executable(test_tiered_abort    
        SOURCES tiered_abort/main.c    
        FLAGS "-DWT_STORAGE_LIB=\"ext/storage_sources/local_store/libwiredtiger_local_store.so\""    
        BINARY_DIR ${CMAKE_CURRENT_BINARY_DIR}/tiered_abort    
    )                                                                                                                                                      
endif()  
```
Commenting out the `define_c_test` block will prevent the test from being compiled. The above change will at least let CMake compile the test (so that we can run it from the command line).
Also might want to add a small comment whilst `define_c_test` is disabled.

Reply:
Done! let me know if that comment describes enough